setup.sh

```diff
@@ -195,3 +187,22 @@ if [ -n "${SETUP_LXD-}" ]; then
         fi
     fi
 fi
+
+if [[ $- == *i* ]]
+then
+  while true
+  do
+    read -p "Do you wish to activate ansible setup in your shell ? "  yn
+    case $yn in
+      y ) if [ ! -f ~/.bashrc ] || ! grep 'source.*activate' ~/.bashrc; then
+            echo '# Activate ansible virtualenv' >> ~/.bashrc
+            echo "source $ANSIBLE_SETUP_DIR/bin/activate" >> ~/.bashrc
+            echo 'export ANSIBLE_STDOUT_CALLBACK=debug' >> ~/.bashrc
+            echo '!! You need to login again for changes to take effect !!'
+          fi
+          break;;
+      n ) exit;;
+      * ) echo "Please answer y or n.";;
+    esac
+  done
+fi
```

Comment:
So in non-interactive mode it doesn't work anymore ?

I'd like to merge this on tuesday morning, so we'll be able to respond if this breaks any project depending on that ;)